paparazzi/build.gradle

```
@@ -73,18 +73,19 @@ dependencies {
   testImplementation libs.truth
 }
 
-tasks.named("dokkaGfm").configure {
-  outputDirectory = rootProject.file("docs/1.x")
+dokka {
+  dokkaPublications.html {
+    outputDirectory.set(rootProject.file("docs/1.x"))
+  }
+
+  dokkaSourceSets.main {
+    reportUndocumented = false
+    skipDeprecated = true
+    jdkVersion = libs.versions.javaTarget.get() as int
 
-  dokkaSourceSets.named("main") {
-    configureEach {
-      reportUndocumented = false
-      skipDeprecated = true
-      jdkVersion = 8
-      perPackageOption {
-        prefix = "app.cash.paparazzi.internal"
-        suppress = true
-      }
+    perPackageOption {
+      matchingRegex = "app.cash.paparazzi.internal.*"
```

`.` is special char in regex: I think the "correct" syntax in Groovy is
``suggestion
      matchingRegex = /app\.cash\.paparazzi\.internal\..*/
```

---

Although it will work most of the time with this pattern, it is still not using periods `.` as proper literals. Meaning any character might match.

This version is most likely the correct one to match `app.cash.paparazzi.internal` and all possible nested packages:
```suggestion
      matchingRegex = /app\.cash\.paparazzi\.internal(\..*)?/
```

@geoff-powell can you rebase your migration or was this PR paused on purpose?